influunt-api/modules/influunt-device/app/os72c/client/virtual/JOptionPaneAnel.java

```diff
@@ -0,0 +1,30 @@
+package os72c.client.virtual;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import javax.swing.*;
+import javax.imageio.ImageIO;
+
+public class JOptionPaneAnel extends GenericJOptionPane implements ActionListener {
+
+    protected JPanel getPanel() {
+        JPanel panel = new JPanel();
+        JLabel label = new JLabel("Selecione o Anel");
+
+        String[] aneis = { "1", "2", "3", "4"};
```

Comment:
O número de anéis depende do que estiver no `ModeloControlador`, inclusive em PROD tem um modelo cadastrado com 12 aneis. Não sei se importa aqui.